tests/test_vitrina.py

```diff
@@ -25,3 +30,45 @@ def test_home(app: DjangoTestApp):
         ['1', 'Organizacijų'],
         ['1', 'Panaudojimo atvejų'],
     ]
+
+@pytest.fixture
+def search_datasets():
+    dataset1 = DatasetFactory(slug='ds1', published=timezone.localize(datetime(2022, 6, 1)))
+    dataset1.set_current_language('en')
+    dataset1.title = 'Dataset 1'
+    dataset1.save()
+    dataset1.set_current_language('lt')
+    dataset1.title = "Duomenų rinkinys vienas"
+    dataset1.save()
+
+    dataset2 = DatasetFactory(slug='ds2', published=timezone.localize(datetime(2022, 8, 1)))
+    dataset2.set_current_language('en')
+    dataset2.title = 'Dataset 2'
+    dataset2.save()
+    dataset2.set_current_language('lt')
+    dataset2.title = "Duomenų rinkinys du\"<'>\\"
+    dataset2.save()
+
+    dataset3 = DatasetFactory(slug='ds3', published=timezone.localize(datetime(2022, 7, 1)))
+    dataset3.set_current_language('en')
+    dataset3.title = 'Dataset 3'
+    dataset3.save()
+    dataset3.set_current_language('lt')
+    dataset3.title = "Duomenų rinkinys trys"
+    dataset3.save()
+    return [dataset1, dataset2, dataset3]
+
+@pytest.mark.haystack
+@pytest.mark.django_db
+def test_empty_search_field(app: DjangoTestApp, search_datasets):
+    form = app.get("/").forms['quick-search']
+    resp = form.submit()
+    assert [int(obj.pk) for obj in resp.context['object_list']] == [search_datasets[1].pk, search_datasets[2].pk, search_datasets[0].pk]
```

Comment:
Always sort results by a reliable sort key and it is better to check titles, instead of list index numbers. Tests like this usually end up failing for random reasons.